modules/webbrowser/src/processors/webbrowserprocessor.cpp

```diff
@@ -266,6 +266,11 @@ void WebBrowserProcessor::OnLoadingStateChange(CefRefPtr<CefBrowser> browser, bo
 }
 
 void WebBrowserProcessor::process() {
+    if (!isBrowserLoading_ && runJSrequested_) {        
```

Comment:
No need to check isBrowserLoading, see isReady in constructor. Also, would not runJS_.isModified() work?

Reply:
when relying on the isReady state, isModified should be sufficient.

Reply:
fixed in latest commit